codecov.yml

```diff
@@ -47,6 +47,6 @@ coverage:
         if_ci_failed: error
 
 comment:
-  layout: "header, diff, changes, sunburst, suggestions, tree"
```

Comment:
Limiting the amount of information being posted into PR comments - just the changes in the files being added/removed/modified are interesting for these.  If files aren't being touched in this PR then the percentage change is mostly meaningless.